### PR TITLE
Update examples for use in benchmarking

### PR DESCRIPTION
**Pull request by tskisner**
This PR greatly simplifies the generation of toast examples.  Instead of directories parameter files and a mix of cmake and bash, we now have a single TOML config file and a python script.

This PR is still a work in progress.  The old files are still in place and the new setup is still be expanded and tested on cori.

### CODE REVIEW

**Comment by tskisner** [comment]
Ok, the generation of the example jobs is working well.  However, we definitely still need to tweak options in `config.toml`.  I suggest we merge this branch now, so that @keskitalo can use these for testing the example script updates.  After those updates, we can have a separate PR to change options in `config.toml` (which may have changed anyway after the work on the example pipelines).

Note:  I did make some small changes to the example pipelines in this branch, but those may be irrelevant (or conflicting with) the changes that @keskitalo is making and should be removed as needed.

File: examples/config.toml [hunk 1]
```diff
@@ -0,0 +1,546 @@
+
```
Comment (keskitalo) [inline examples/config.toml:1]: This is a really clean way of defining the example jobs.

File: examples/generate.py [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+
+    outstr = "{}\n".format(outstr)
+
+    return outstr
+
+
+def ceil_div(a, b):
+    """Handy, from: https://stackoverflow.com/questions/14822184/
+    """
+    return -(-a // b)
```
Comment (keskitalo) [inline examples/generate.py:114]: Where was this function all my life?

File: examples/trigger_astropy.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```
Comment (keskitalo) [inline examples/trigger_astropy.py:1]: This script should deploy with `healpy`. Have you proposed it?

Comment (tskisner) [reply examples/trigger_astropy.py:1]: An interesting idea.  Perhaps someday...

**Pull request merged**